src/main/scripts/cd/external_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+
+import requests
+from urllib.parse import urljoin
+import json
+import time
+import sys
+import os
+
+class TravisBuilds:
+    """
+    This class provides requests to builds and check their statuses
+    """
+    travis_api_url = 'https://api.travis-ci.com/'
```

Comment:
check if it is possible to do it with travis-ci.org (scalecube is in org)

Reply:
It's possible